examples/energy-management-app/energy-management-common/src/EVSEManufacturerImpl.cpp

```diff
@@ -104,6 +107,224 @@ CHIP_ERROR EVSEManufacturer::Shutdown()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FindNextTarget(const uint8_t dayOfWeekMap, uint16_t minutesPastMidnightNow_m, uint16_t & targetTimeMinutesPastMidnight_m,
+                          DataModel::Nullable<Percent> & targetSoC, DataModel::Nullable<int64_t> & addedEnergy_mWh,
+                          bool bAllowTargetsInPast)
+{
+
+    CHIP_ERROR err = CHIP_NO_ERROR;
```

Comment:
This style of code makes the code hard to follow: the final `return err` depends on the initialization that happens at the start of the function and following to see if anything in the code changes it.

Please remove this declaration and do a explicit `return bFound ? CHIP_NO_ERROR : CHIP_ERROR_NOT_FOUND` or similar.

Reply:
Change made.